config/options.go

```diff
@@ -561,6 +565,11 @@ func (o *Options) Validate() error {
 			return fmt.Errorf("config: bad forward-auth-url %s : %w", o.ForwardAuthURLString, err)
 		}
 		o.ForwardAuthURL = u
+		switch o.ForwardAuthType {
+		case ProxyTypeNginx, ProxyTypeTraefik:
+		default:
+			return fmt.Errorf("config: bad forward-auth-type: %s, supported types: %v", o.ForwardAuthType, ProxyTypes)
```

Comment:
what about `forward-auth-type`: %s is required when `forward-auth-url` is set, supported types: %v", o.ForwardAuthType, ProxyTypes)

Reply:
Fixed